common/boot_command/boot_command_ast.go

```diff
@@ -0,0 +1,123 @@
+package bootcommand
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"strings"
+	"time"
+)
+
+/*
+TODO:
+	* pc-at abstraction
```

Comment:
pc-xt?

Reply:
yeah missed this in a2cc17f, but since this is a todo comment I'll let it fall off when it gets done